botorch/utils/sampling.py

```diff
@@ -838,6 +861,7 @@ def get_polytope_samples(
     seed: Optional[int] = None,
     thinning: int = 32,
     n_burnin: int = 10_000,
+    normalize_bounds: bool = True,
```

Comment:
Let's just do this by default and get rid of the option to simplify he interface? I don't really see a benefit for not doing this, worst case there is a bit of overhead but that's mostly negligible compared to the cost of drawing the samples.

Reply:
I agree, I removed the keyword.